packages/react-storage/src/components/StorageBrowser/actions/handlers/__tests__/delete.spec.ts

```diff
@@ -6,22 +6,24 @@ const removeSpy = jest.spyOn(StorageModule, 'remove');
 
 const baseInput: DeleteHandlerInput = {
   prefix: 'prefix/',
-  key: 'key',
+  key: 'prefix/key',
   config: {
     accountId: '012345678901',
     bucket: 'bucket',
     credentials: jest.fn(),
     region: 'region',
   },
-  data: { key: 'key' },
+  // @ts-expect-error
+  // FIXME: The type for payload is never
+  data: { id: 'id', payload: undefined },
```

Comment:
Leaving this error here per conversation with @calebpollman because there is ongoing work to refactor the data type